packages/graphql-auth-transformer/src/resources.ts

```diff
@@ -138,21 +139,29 @@ export class ResourceFactory {
             return comment(`No Static Group Authorization Rules`)
         }
         const allowedGroups: string[] = []
+        let customClaim: string;
         for (const rule of rules) {
             const groups = rule.groups;
             for (const group of groups) {
                 if (group) {
                     allowedGroups.push(group);
                 }
             }
+            if (rule.groupClaim) {
+                if (customClaim) {
+                    throw new InvalidDirectiveError('@auth directive currently only supports one source for groupClaim!')
```

Comment:
format: no need for exclamation mark, also we should output what type, what directive, what claim we found, what extra claim we found, that helps the developers to locate the error easier.